### ISSUE TITLE
Plugin build fails due to IEncryptionManager

### ISSUE THREAD

**Issue by joshuaboniface** [jellyfin/jellyfin-plugin-emailnotifications#5]
```
Api/ServerApiEntryPoints.cs(22,26): error CS0246: The type or namespace name 'IEncryptionManager' could not be found (are you missing a using directive or an assembly reference?) [/srv/jellyfin/projects/plugin/jellyfin-plugin-emailnotifications/MediaBrowser.Plugins.SmtpNotifications/MediaBrowser.Plugins.SmtpNotifications.csproj]
Api/ServerApiEntryPoints.cs(25,77): error CS0246: The type or namespace name 'IEncryptionManager' could not be found (are you missing a using directive or an assembly reference?) [/srv/jellyfin/projects/plugin/jellyfin-plugin-emailnotifications/MediaBrowser.Plugins.SmtpNotifications/MediaBrowser.Plugins.SmtpNotifications.csproj]
Notifier.cs(17,26): error CS0246: The type or namespace name 'IEncryptionManager' could not be found (are you missing a using directive or an assembly reference?) [/srv/jellyfin/projects/plugin/jellyfin-plugin-emailnotifications/MediaBrowser.Plugins.SmtpNotifications/MediaBrowser.Plugins.SmtpNotifications.csproj]
Notifier.cs(21,41): error CS0246: The type or namespace name 'IEncryptionManager' could not be found (are you missing a using directive or an assembly reference?) [/srv/jellyfin/projects/plugin/jellyfin-plugin-emailnotifications/MediaBrowser.Plugins.SmtpNotifications/MediaBrowser.Plugins.SmtpNotifications.csproj]
Plugin.cs(19,26): error CS0246: The type or namespace name 'IEncryptionManager' could not be found (are you missing a using directive or an assembly reference?) [/srv/jellyfin/projects/plugin/jellyfin-plugin-emailnotifications/MediaBrowser.Plugins.SmtpNotifications/MediaBrowser.Plugins.SmtpNotifications.csproj]
Plugin.cs(21,89): error CS0246: The type or namespace name 'IEncryptionManager' could not be found (are you missing a using directive or an assembly reference?) [/srv/jellyfin/projects/plugin/jellyfin-plugin-emailnotifications/MediaBrowser.Plugins.SmtpNotifications/MediaBrowser.Plugins.SmtpNotifications.csproj]
```

**Comment by joshuaboniface** [comment]
Related to #4

**Comment by dkanada** [comment]
fixed with #6 and #7
